web/src/user_group_edit_members.ts

```diff
@@ -179,32 +180,115 @@ export function rerender_members_list({
     });
 }
 
+function generate_group_link(group: UserGroup): string {
+    if (group.is_system_group) {
+        let group_name = group.name.split(":")[1];
+        if (group_name) {
+            group_name = group_name?.charAt(0).toUpperCase() + group_name?.slice(1);
+            return `<a data-user-group-id="${group.id}" class="view_user_group">${$t({defaultMessage: "Role"})}:${group_name}</a>`;
```

Comment:
I think we can use `user_groups.get_display_group_name` here, it should be the same names as shown in typeahead.